### DISCUSSION TITLE
Local LLM fine tune prompt

### DISCUSSION THREAD

**Discussion by fis60** [SubtitleEdit/subtitleedit#8300] [General]
Hi am using Auto-translate with Ollama Model llama3 but it skips a line or merge some texts and the timing of the whole subtitle becomes wrong also it adds Here is your translation for some sentences any way to add prompt's to the translations?

**Comment by dgoryeo** [comment]
@fis60 , I'm keen to hear your experience with llama 3, how do you rate the quality of translations --say compared to GPT3.5? Thanks

**Comment by fis60** [comment]
@dgoryeo The translation it self is actually great way better than google or any other services compared to gpt3.5 it's up there but gpt3.5 is still a bit better keep in mind i use 8b parameters only and it's very fast using my local pc
